flink-table/flink-sql-gateway/src/main/java/org/apache/flink/table/gateway/service/operation/OperationExecutor.java

```diff
@@ -104,8 +105,15 @@ public Set<String> listCatalogs() {
         return getTableEnvironment().getCatalogManager().listCatalogs();
     }
 
+
+    public Set<String> listDatabases(String catalogName) {
+        return Collections.unmodifiableSet(
+                new HashSet<>(getTableEnvironment().getCatalogManager().listSchemas(catalogName)));
```

Comment:
listSchema is different from the listDatabases. `listSchemas` also includes the temporary object here but `listDatabases` doesn't.  We should keep the meaning and the actual implementation is the same here.

Reply:
Fixed.